src/Raven.Server/Documents/Handlers/Debugging/MemoryDebugHandler.cs

```diff
@@ -190,14 +190,16 @@ public Task MemorySmaps()
         [RavenAction("/admin/debug/memory/stats", "GET", AuthorizationStatus.Operator, IsDebugInformationEndpoint = true)]
```

Comment:
This is a debug information endpoint, it means that we will not include threads ans mappings for debug package. Is taht what we want? cc @karmeli87

Reply:
I changed it to be `true` by default.